1.8.3/amd64/debian/entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 set -euo pipefail
```

Comment:
Because entrypoint.sh uses "set -euo pipefail" it will probably still exit as soon as any command returns a non-zero exit code. See: https://explainshell.com/explain?cmd=set+-euo+pipefail

Reply:
OK, how can I prevent that?